diffpy/pdfmorph/pdfmorphapp.py

```diff
@@ -412,7 +468,178 @@ def main():
         )
 
-    return
+    return morph_results
+
+
+def multiple_morphs(parser, opts, pargs, stdout_flag=True):
+    # Custom error messages since usage is distinct when --multiple tag is applied
+    if len(pargs) < 2:
+        parser.custom_error("You must supply FILE and DIRECTORY. Go to --help for usage.")
+    elif len(pargs) > 2:
+        parser.custom_error("Too many arguments. Go to --help for usage.")
+
+    # Parse paths
+    morph_file = Path(pargs[0])
+    if not morph_file.is_file():
+        parser.custom_error(f"{morph_file} is not a file. Go to --help for usage.")
+    target_directory = Path(pargs[1])
+    if not target_directory.is_dir():
+        parser.custom_error(f"{target_directory} is not a directory. Go to --help for usage.")
+
+    # Do not morph morph_file against itself if it is in the same directory
+    target_list = list(target_directory.iterdir())
+    if morph_file in target_list:
+        target_list.remove(morph_file)
+
+    # Format field name for printing and plotting
+    field = None
+    if opts.field is not None:
+        field_words = opts.field.split()
+        field = ""
+        for word in field_words:
+            field += f"{word[0].upper()}{word[1:].lower()}"
+    field_list = None
+
+    # Sort files in directory by some field
+    if field is not None:
+        try:
+            target_list, field_list = tools.field_sort(target_list, field, opts.reverse, opts.serfile,
+                                                       get_field_values=True)
+        except KeyError:
+            if opts.serfile is not None:
+                parser.custom_error("The requested field was not found in the metadata file.")
+            else:
+                parser.custom_error("The requested field is missing from a PDF file header.")
+    else:
+        # Default is alphabetical sort
+        target_list.sort(reverse=opts.reverse)
+
+    # Disable single morph plotting
+    plot_opt = opts.plot
+    opts.plot = False
+
+    # Manage saving
+    save_opt = opts.saveloc
+    save_names = {}
+    save_morphs_here = ""
+    if save_opt is not None:
+        try:
+            # Make directory to save files in if it does not already exist
+            Path(save_opt).mkdir(parents=True, exist_ok=True)
+
+            # Morphs will be saved in the subdirectory "Morphs"
+            save_morphs_here = Path(save_opt).joinpath("Morphs")
+            save_morphs_here.mkdir(exist_ok=True)
+
+            # Get names for the saved morphs
+            if opts.snfile is not None:
+                # Names should be stored properly in opts.snfile
+                save_names = tools.deserialize(opts.snfile)
+            # Default naming scheme
+            else:
+                for target_file in target_list:
+                    save_names.update({target_file.name: {"save_morph_as":
+                                                          f"Morph_with_Target_{target_file.stem}.cgr"}})
```

Comment:
The `-s` option for multiple morphs saves all the morphs to a user-specified directory. Default names for the names of morphs are "Morph_with_Target_{target_PDF_name}.cgr"